pkg/auth/handler/webapp/authflow_controller.go

```diff
@@ -194,7 +194,7 @@ func (c *AuthflowController) HandleOAuthCallback(w http.ResponseWriter, r *http.
 		return
 	}
 
-	s, err := c.getWebSession(r)
+	s, err := c.Sessions.Get(state.WebSessionID)
```

Comment:
The session should be in cookie, is there any reason we need to get it from the state?
I remembered that WebSessionID is for wechat login.